lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -635,10 +593,12 @@ def _install_service(self) -> None:
         Raises:
             SlurmOpsError: Raised if `apt` fails to install the required Slurm packages.
         """
-        packages = [self._service_name, "munge", "mungectl", "prometheus-slurm-exporter"]
+        packages = [self._service_name, "munge", "mungectl"]
         match self._service_name:
+            case "sackd":
+                packages.extend(["slurm-client"])
```

Comment:
Also "libpmix-dev" and "openmpi-bin" to let users build MPI codes and to keep the environment consistent with the compute nodes?

Reply:
Hey hey, libpmix-dev is a runtime dependency for slurmd and slurmctld. Openmpi-bin is added to compute nodes so they can run mpi workloads. I don't think either of these packages belong on the login node.

Reply:
Hello! Definitely agreed MPI workloads shouldn't run on the login node. My thought process for having OpenMPI was to allow users to compile MPI binaries for use on the compute nodes since that's commonly done on login nodes (though it'd need "libopenmpi-dev" since "openmpi-bin" provides only `mpicc`, etc. compiler wrappers and not `mpi.h`).

Maybe worth revisiting in a future PR. Happy to keep this one focused on `SackdManager` and stick with "slurm-client".

Reply:
For now we should have `slurm-client` be the only additional package on the login node.

When we put together some howtos for common HPC use cases such as how to submit jobs, that would be a great time to identify other packages we should be shipping on the nodes. Our software stack story is also still loosely defined, so it might not even be the `sackd` charm itself providing the packages folks will load and use.

Reply:
Only installing `slurm-client` with `sackd` for now. Will see what else should be added to the nodes once we dive more into the "providing dependencies for user workloads" story.